### PR TITLE
Update lang-PL.yml

### PR DESCRIPTION
**Pull request by kacperleague9**

### CODE REVIEW

File: advanced-achievements-plugin/src/main/resources/lang-PL.yml [hunk 1]
```diff
@@ -6,207 +7,221 @@
 # |     untranslated parameters to the end of the file.     | #
```
Comment (PyvesB) [inline advanced-achievements-plugin/src/main/resources/lang-PL.yml]: Could you please bump the version number up here to 5.15.1?

File: advanced-achievements-plugin/src/main/resources/lang-PL.yml [hunk 2]
```diff
@@ -6,207 +7,221 @@
...
 # Related to /aach inspect
-achievement-not-recognized: 'Achievement called NAME was not recognized. Did you mean CLOSEST_MATCH?'
+achievement-not-recognized: Nie odnaleziono osiagniecia. Czy chodzilo ci o CLOSEST_MATCH?
 
 #============================================================#
 # +--------------------------------------------------------+ #
 # | Parameters in future plugin versions will  appear here | #
 # +--------------------------------------------------------+ #
 #============================================================#
+list-raids-won: '&6Wygrane Raidy'
```
Comment (PyvesB) [inline advanced-achievements-plugin/src/main/resources/lang-PL.yml]: Could this parameters please be moved up with the other related `list` ones?

**Comment by kacperleague9** [comment]
Will complete them asap

**Comment by PyvesB** [comment]
@kacperleague9 any chance we could get this landed? 😉

**Pull request merged**